sql/moz-fx-data-marketing-prod/ga_derived/www_site_downloads_v2/query.sql

```diff
@@ -47,4 +101,13 @@ GROUP BY
   medium,
   campaign,
   ad_content,
-  browser
+  browser,
+  download_events,
+  download_events AS downloads,
+  CASE
```

Comment:
nit: since this is true or false I generally prefer using `if` and reserve the use of `CASE` for cases with more than 2 possible outcomes.